libs/ui-lib/lib/ocm/components/clusterConfiguration/staticIp/components/FormViewHosts/formViewHostsValidationSchema.tsx

```diff
@@ -63,6 +75,18 @@ const getHostValidationSchema = (networkWideValues: FormViewNetworkWideValues) =
             )
         : Yup.string(),
     }),
+    bondPrimaryInterface: Yup.mixed().when('useBond', {
+      is: true,
+      then: () =>
+        macAddressValidationSchema.concat(getUniqueValidationSchema(getAllBondPrimaryInterfaces)),
+      otherwise: () => Yup.mixed().notRequired(),
+    }),
+    bondSecondaryInterface: Yup.mixed().when('useBond', {
+      is: true,
+      then: () =>
+        macAddressValidationSchema.concat(getUniqueValidationSchema(getAllSecondaryInterfaces)),
+      otherwise: () => Yup.mixed().notRequired(),
+    }),
```

Comment:
It seems you got the unique validations the wrong way around.

Primary and secondary interfaces in one host cannot be the same. A different host is allowed to have interfaces with the same MAC address values, it seems.

Also, MAC addresses are case insensitive which the check currently doesn't implement. (Neither does the BE but I think that's a bug.)